x64dbg-mcp/x64dbg_mcp/main.py

```diff
@@ -184,12 +184,13 @@ def create_dbg_client(exe:PE):
 
     logger.info(f"run {target} {cmdline}")
     dbgClient.start_session(target_exe=target, cmdline=cmdline)
-    if not dbgClient.go():
-        raise Exception(f"can not launch {target} {cmdline}")
     dbgClient.wait_until_debugging(99999) # make sure item is in debuggable state
     dbgClient.clear_breakpoint()
+
+    continue_execution() # magic, i don't know why this worked :P
```

Comment:
It in fact does not work 😅 I had to remove this locally for the debuggee to launch correctly and land at the system breakpoint.